cmd/image/qcow2ova/prep/prepare.go

```diff
@@ -83,6 +83,36 @@ func prepare(mnt, volume, dist, rhnuser, rhnpasswd, rootpasswd string) error {
 		return fmt.Errorf("unable to handle the %s filesystem for %s", fsType, partDev)
 	}
 
+	file := filepath.Join(mnt, "etc/fstab")
+
+	//get the boot partition name
+	deviceuuid, err := bootDeviceuuid(file)
+	if err != nil {
+		return err
+	}
+
+	if deviceuuid != "" {
+		uuid := strings.Split(deviceuuid, "=")[1]
+		bootDev, err := findDevice(uuid)
+		if err != nil {
+			return err
+		}
+		err = mount("nouuid", bootDev, filepath.Join(mnt+"/boot"))
+		if err != nil {
+			return err
+		}
```

Comment:
Can inline `if` statement
```suggestion
		if err = mount("nouuid", bootDev, filepath.Join(mnt+"/boot")); err != nil {
			return err
		}
```

Reply:
IMO, adding an inline if statement makes it longer, I think it looks better with multiple lines.

Reply:
Leaving it to your choice. 👍